cmd/docker/main.go

```diff
@@ -46,7 +46,7 @@ type CliOptions struct {
 	proto         string
 }
 
-const productName = "vSphere Integrated Containers"
+const productName = "vSphere Integrated Containers 0.3"
```

Comment:
I have a GI for changing the way we do strings in the personality server.  I'd rather do that there than in this PR.